news/823.new.rst

```diff
@@ -1,2 +1,4 @@
 Add hook for ``numbers_parser`` to ensure that package's data file is
 collected.
+
+Add hook for ``sv_ttk`` to ensure that files are collected.
```

Comment:
Sorry, missed this one earlier. The news entry for this PR should go into a separate file, `news/826.new.rst`.

Reply:
Ups. Now I see where I need the pull request id. Will be changed :)